doc/nrf/libraries/bluetooth/services/rrsp.rst

```diff
@@ -0,0 +1,59 @@
+.. _rrsp_readme:
+
+Ranging Responder (RRSP)
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+This library implements the Ranging Responder for Channel Sounding with the corresponding set of characteristics defined in the `Ranging Service Specification`_ and the `Ranging Profile Specification`_.
+
+Supported features
+==================
+
+- On Demand Ranging Data
+
+Configuration
```

Comment:
Do the same adjustments as in the other library doc. This PR also needs a changelog entry.